source/particle/property/elastic_stress.cc

```diff
@@ -121,7 +121,7 @@ namespace aspect
         this->get_material_model().evaluate (material_inputs,material_outputs);
 
         for (unsigned int i = 0; i < SymmetricTensor<2,dim>::n_independent_components ; ++i)
-          particle->get_properties()[data_position + i] += material_outputs.reaction_terms[0][i];
+          particle->get_properties()[data_position + i] = material_inputs.composition[0][i] + material_outputs.reaction_terms[0][i];
```

Comment:
This is potentially a problem. By first interpolating the particle property to the field and then assigning back to the particle we introduce a lot of numerical diffusion. Why does the approach above not work? Because of the nonlinear solver?

Reply:
I think my reasoning was as follows:
* In the operator splitting step, we update the solution (the stress compositional fields) of the previous timestep.
* After that we advance the particles and evaluate the updated solution at their location
* We then update the particle properties, in this case the stress. If we use the stress values on the particles, then these do not include the operator splitting update.